src/crypto/chacha20.h

```diff
@@ -20,7 +20,9 @@ class ChaCha20
     void SetKey(const unsigned char* key, size_t keylen);
     void SetIV(uint64_t iv);
     void Seek(uint64_t pos);
-    void Output(unsigned char* output, size_t bytes);
+
+    // if <input> is provided, the ciphertext will be written to <output> otherwise the current keystream
```

Comment:
This comment seems imply that output is not written if input is null. Maybe rephrase: "If \<input> is non-null, the specified number of \<bytes> from \<input> will encrypted and written to \<output>, otherwise just the keystream will be written to \<output\> (which may be useful for random number generation). Either way, the number of bytes written to \<output> will be \<bytes>."

Reply:
Valid point. Fixed.